api/src/utils/aggregate-utils.ts

```diff
@@ -0,0 +1,79 @@
+import { IPitch, IUser, IUserAggregate, IPitchAggregate } from 'ssw-common';
+
+import Pitch, { PitchSchema } from '../models/pitch';
+import User, { UserSchema } from '../models/user';
+
+import { santitizePitch, santitizeUser } from './helpers';
+
+const simplifyUser = (user: IUser): Partial<IUser> => ({
+  firstName: user.firstName,
+  preferredName: user.preferredName,
+  lastName: user.lastName,
+  profilePic: user.profilePic,
+  _id: user._id,
+});
+
+const aggregatePitch = async (
+  rawPitch: PitchSchema,
+): Promise<IPitchAggregate> => {
+  const author = simplifyUser(await User.findById(rawPitch.author));
+  const reviewer = simplifyUser(await User.findById(rawPitch.approvedBy));
+
+  const assignmentContributors = await Promise.all(
+    rawPitch.assignmentContributors.map(async (contributor) => ({
+      user: simplifyUser(await User.findById(contributor.userId)),
+      teams: contributor.teams,
+    })),
+  );
+
+  const pendingContributors = await Promise.all(
+    rawPitch.pendingContributors.map(async (contributor) => ({
+      user: simplifyUser(await User.findById(contributor.userId)),
+      teams: contributor.teams,
+    })),
+  );
```

Comment:
Can these awaits be Promise.all()'d rather than being awaited one after another since they're independent.